### ISSUE TITLE
Arrow navigation for radio buttons is caught on map panning

### ISSUE THREAD

**Issue by warm-coolguy** [Dataport/polar#253]
When navigating via keyboard to e.g. the LayerChooser, radio buttons can no longer be used by the ArrowUp/ArrowDown input. Instead, the map is panned.

Looking at the Meldemichel, this seems to be a relatively new issue.

- Works in last deployment: https://static.hamburg.de/kartenclient/prod/
- Doesn't work in current dev build: https://dataport.github.io/polar/docs/meldemichel/example/complete.html

**Comment by warm-coolguy** [comment]
Broken since https://github.com/Dataport/polar/commit/6a1ca4d31ee5da3566fa0481e6f3b181398d1c4f. Since that didn't change event handling, it's probably from some change in either the masterportalAPI or ol itself.

**Comment by warm-coolguy** [comment]
I suppose https://github.com/openlayers/openlayers/blob/main/src/ol/interaction/KeyboardPan.js#L86 handling the event in favour of map panning and stopping others from consuming the event is the root cause. This can be handled on our side, PR on the way.
